Gemfile

```diff
@@ -25,6 +25,8 @@ end
 
 gem 'plek', '1.7.0'
 
+gem 'govuk-content-schema-test-helpers', '~> 1.1.0'
```

Comment:
Thanks - this was temporarily used in development but backed out later. We'll move this to the test group, and add schema tests, in a subsequent PR.